src/set_theory/ordinal/basic.lean

```diff
@@ -729,8 +704,8 @@ def typein.principal_seg {α : Type u} (r : α → α → Prop) [is_well_order 
 
 /-- The cardinal of an ordinal is the cardinal of any
   set with that order type. -/
-def card (o : ordinal) : cardinal :=
-quot.lift_on o (λ a, #a.α) $ λ ⟨α, r, _⟩ ⟨β, s, _⟩ ⟨e⟩, quotient.sound ⟨e.to_equiv⟩
+def card : ordinal → cardinal :=
+quotient.map Well_order.α $ λ ⟨α, r, _⟩ ⟨β, s, _⟩ ⟨e⟩, ⟨e.to_equiv⟩
```

Comment:
This is def-eq to the previous definition, right?

Reply:
Yes.